spot_driver/launch/spot_driver.launch.py

```diff
@@ -106,6 +113,19 @@ def launch_setup(context: LaunchContext, ld: LaunchDescription) -> None:
 
     ld.add_action(rviz)
 
+    registered_depth_image_nodes = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [FindPackageShare(THIS_PACKAGE), "launch", "spot_depth_registered_publishers.launch.py"]
+                )
+            ]
+        ),
+        launch_arguments={"spot_name": spot_name, "has_arm": has_arm}.items(),
+        condition=IfCondition(use_depth_registered_nodelets),
```

Comment:
If this is check on if `use_depth_registered_nodelets == "False" or ues_depth_registered_nodeltes == "false"`, it would nice to use this for `has_arm` instead of the current workaround.